### ISSUE TITLE
Training Issue

### ISSUE THREAD

**Issue by Inbalc2** [computational-cell-analytics/micro-sam#712]
Hi,
I have a dataset with binary masks. I am tryting to run the code in finetune_hela.py but it expects the ground truth to be 4 channels. What are those 4 channels and how can I train with masks with only 2 channels? 
Thanks

**Comment by constantinpape** [comment]
This seems to be resolved. Please reopen if you have further questions.
